src/sql/workbench/contrib/notebook/test/browser/notebookMarkdown.test.ts

```diff
@@ -53,12 +54,19 @@ suite('NotebookMarkdownRenderer', () => {
 	});
 
 	test('link from relative file path', () => {
-		notebookMarkdownRenderer.setNotebookURI(URI.parse('maddy/temp/file1.txt'));
+		notebookMarkdownRenderer.setNotebookURI(URI.parse('foo/temp/file1.txt'));
 		let result: HTMLElement = notebookMarkdownRenderer.renderMarkdown({ value: `[Link to relative path](../test/.build/someimageurl)`, isTrusted: true });
 		if (process.platform === 'win32') {
-			assert.strictEqual(result.innerHTML, `<p><a href="\\maddy\\test\\.build\\someimageurl" data-href="\\maddy\\test\\.build\\someimageurl" title="\\maddy\\test\\.build\\someimageurl">Link to relative path</a></p>`);
+			let fullPath = path.resolve('.');
```

Comment:
I'm not super clear what this is doing. Would the second condition only be true if the enlistment was run from a UNC path? 

I think the test should ideally be just testing if we have a UNC path in the notebook, if WYSIWYG mucks with the math unnecessarily or not (i.e. doesn't have anything to do with what the relative path to the notebook is).

Reply:
So, if we need to change this test, then I'm a bit worried that we're changing the implementation of how UNC paths are being handled.